grtc/DESIGN.md

```diff
@@ -0,0 +1,193 @@
+# Design
```

Comment:
Can we call this "architecture" to avoid clashes with design docs on searches, etc?

Reply:
Should we keep grtc in its own package?

Reply:
It's much smaller after encryption got removed. If we would keep it, where should we put it in the repository? Can it remain in the top level or rather should be moved to `editor` top level folder?